{"name": "Проверка получения заказа с несуществующим номером (треком)", "status": "passed", "start": 1725209603505, "stop": 1725209603658, "uuid": "a619a6ed-5fd4-4458-944c-92be3721e781", "historyId": "3d3f2b64d5b60c587fd321c4689198b4", "testCaseId": "3d3f2b64d5b60c587fd321c4689198b4", "fullName": "tests.test_get_order_to_track_endpoint.TestGetOrderToTrackEndpoint#test_get_order_to_track_false_order", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order_to_track_endpoint"}, {"name": "subSuite", "value": "TestGetOrderToTrackEndpoint"}, {"name": "host", "value": "DNn"}, {"name": "thread", "value": "17452-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order_to_track_endpoint"}]}